docs/guides/admin/docs/workflowoperationhandlers/amberscript-start-transcription-woh.md

```diff
@@ -0,0 +1,15 @@
+# AmberScript Start Transcription
+
+## Description
+
+AmberScript Start Transcription invokes the AmberScript Transcription Service by submitting
+an audio or video file to be transcribed and captioned.
+
+## Parameter Table
+
+| configuration keys    | description                                                              | default      | example            |
+|-----------------------|--------------------------------------------------------------------------|--------------|--------------------|
+| source-tag            | A tag selecting the audio or video file to be sent for translation.      | transcript   | engage-download    |
+| jobtype               | direct or perfect                                                        | direct       | perfect            |
```

Comment:
What is the jobtype for? It will be useful in the documentation to define what it is or what it does.

Reply:
fixed